__tests__/app.test.js

```diff
@@ -264,6 +264,29 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("GET /api/users", () => {
+  test("200:should respond with the correct length of the users array", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length).toBe(4);
+      });
```

Comment:
We could potentially have both of these tests as one, but up to you if you want to refactor! 😄